tests/Xamarin.Forms-Performance-Integration/Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="Xamarin.Forms_Performance_Integration">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.formsperformanceintegration">
```

Comment:
Do we need to change the package name?

If so, the `Activity` name, `_XamarinFormsIntergationPackage` and `TestApk` need update: https://github.com/xamarin/xamarin-android/blob/master/tests/Xamarin.Forms-Performance-Integration/Droid/Xamarin.Forms.Performance.Integration.Droid.projitems#L9-L17